File: sumatoria_multiplicacion.py
def operacion_suma(a, b, c):
    Resultado_Suma = a + b + c
    print("El resultado de la suma es :", Resultado_Suma)
    return Resultado_Suma


def operacion_multiplicacion(aa, bb, cc):
    Resultado_Producto = aa * bb * cc
    print("El resultado de la multiplicacion es :", Resultado_Producto)
    return Resultado_Producto


def main():
    Numero1 = int(input("Ingrese Numero 1:\n"))
    Numero2 = int(input("Ingrese Numero 2:\n"))
    Numero3 = int(input("Ingrese Numero 3:\n"))

    if (Numero1 > 0):
        resulMul = operacion_multiplicacion(Numero1, Numero2, Numero3)
    else:
        resulSum = operacion_suma(Numero1, Numero2, Numero3)


if __name__ == "__main__":
    main()

File: sumatoria_multiplicacion.cpp
#include <cstdlib>
#include <iostream>

using namespace std;

int Suma(int a, int b, int c){
    int resultado;
    resultado = a + b + c;
    cout << "La suma de los numeros es: " << resultado << endl;
    return resultado;
}
int Multiplicacion(int numero1, int numero2, int numero3){
    int resultado;
    resultado = numero1 * numero2 * numero3;
    cout << "El producto de los numeros es: " << resultado << endl;
    return resultado;
}
int main(){
    int Numero1, Numero2, Numero3;
    int Resultado_Multiplicacion, Resultado_Suma;
    cout << "Ingrese los valores para los numeros: " << endl;
    cin >> Numero1 >> Numero2 >> Numero3;

    if(Numero1 > 0){
        Resultado_Multiplicacion = Multiplicacion(Numero1, Numero2, Numero3);
    } else {
        Resultado_Suma = Suma(Numero1, Numero2, Numero3);
    }
    return 0;
}
